pages/common/keepassxc-cli.md

```diff
@@ -0,0 +1,36 @@
+# keepassxc-cli
+
+> Command-line interface for KeepassXC.
+> More information: <https://keepassxc.org/>.
+
+- Search entries:
+
+`keepassxc-cli lookup {{path/to/database_file}} {{name}}`
+
+- List the contents a folder:
+
+`keepassxc-cli ls {{path/to/database_file}} {{/path/to/folder_name}}`
```

Comment:
```suggestion
`keepassxc-cli ls {{path/to/database_file}} {{path/to/folder_name}}`
```
Does the folder path have to be absolute?

Reply:
It does

Reply:
why `folder_name`? sounds redundant because every path use the folder/directory name.

Reply:
Why not just `path/to/directory` like we use everywhere?
```suggestion
`keepassxc-cli ls {{path/to/database_file}} {{/path/to/directory}}`
```